clusters/hive/hypershift/README.md

```diff
@@ -0,0 +1,21 @@
+This directory includes the files (include generated files) for deploying HyperShift on `hive`.
+
+# HyperShift CLI and operator images
+HyperShift CLI (the utility for installing HyperShift, as well as creting/destroying hosted clusters) and HyperShift operator (the core component on HyperShift management cluster) share the same image, due to the [fact](https://issues.redhat.com/browse/HOSTEDCP-792) that HyperShift does not provide backward or forward compatibilities.
+
+The image is built with `clusters/app.ci/supplemental-ci-images/hypershift/hypershift-cli.yaml` to available for all build farms.
+
+# Deployment/upgrading on `hive`
+Prepare the AWS credentials for your management cluster account in `~/workspace/mgmt-aws` (or any other path you prefer).
+
+Generate the deployment file with
+```
+docker run -ti --rm -v ~/workspace/mgmt-aws:/mgmt-aws:ro registry.ci.openshift.org/ci/hypershift-cli:latest install --oidc-storage-provider-s3-bucket-name=hypershift-oidc-provider --oidc-storage-provider-s3-credentials=/mgmt-aws --oidc-storage-provider-s3-region=us-east-1 --hypershift-image=https://registry.ci.openshift.org/ci/hypershift-cli:latest --enable-uwm-telemetry-remote-write=false render
+```
+
+Then apply the file with
+`oc --context hive apply -f hypershift.yaml`
+
+# Note
+!!! IMPORTANT !!!
+The CLI used to generate the deployment file must be the _EXACT_ CLI used to create/destroy hosted cluster.
```

Comment:
- Could you add the command to get the cli?

Reply:
I believe thats already included
```
docker run -ti --rm -v ~/workspace/mgmt-aws:/mgmt-aws:ro registry.ci.openshift.org/ci/hypershift-cli:latest install --oidc-storage-provider-s3-bucket-name=hypershift-oidc-provider --oidc-storage-provider-s3-credentials=/mgmt-aws --oidc-storage-provider-s3-region=us-east-1 --hypershift-image=https://registry.ci.openshift.org/ci/hypershift-cli:latest --enable-uwm-telemetry-remote-write=false render
```

Reply:
The `cli` here is docker, no?